assets/css/style.scss

```diff
@@ -6327,6 +6426,48 @@ Data API CSS
   }
 
   & a {
+  }
+}
+
+.no-margin {
+  margin: 0 !important;
```

Comment:
I typically only use `!important` on what I call "override" classes. I usually avoid them, but it's very difficult to do that when trying to customize existing css, especially because the existing css uses `!important` a lot. The css class hierarchy is pretty good, but once a few `!important`s are in there, it make things more difficult